src/main/java/com/hubspot/jinjava/el/ext/ExtendedParser.java

```diff
@@ -53,6 +64,19 @@
   static final Scanner.ExtensionToken TRUNC_DIV = TruncDivOperator.TOKEN;
   static final Scanner.ExtensionToken POWER_OF = PowerOfOperator.TOKEN;
 
+  static final Set<Symbol> VALID_SYMBOLS_FOR_EXP_TEST = Sets.newHashSet(
+    IDENTIFIER,
+    EQ,
+    NE,
+    LT,
+    LE,
+    GT,
+    GE,
+    TRUE,
+    FALSE,
+    CollectionMembershipOperator.TOKEN.getSymbol()
```

Comment:
CollectionMembershipOperator.TOKEN.getSymbol() is just `EXTENSION` which is non-ideal because we really just want to add `IN` to this set, but we can't add symbols.